pkg/autodiscovery/providers/kube_services.go

```diff
@@ -65,6 +66,13 @@ func NewKubeServiceConfigProvider(*config.ConfigurationProviders) (ConfigProvide
 		return nil, fmt.Errorf("cannot add event handler to services informer: %s", err)
 	}
 
+	if config.Datadog.GetBool("cluster_checks.support_hybrid_ignore_ad_tags") {
+		log.Warnf("The `cluster_checks.support_hybrid_ignore_ad_tags` flag is" +
```

Comment:
can we move this log warning in the config package to have it only at start time?

Reply:
It should be good where it is as it's not a hot path 👍

Reply:
no it the hot path so it should be good.